src/routers/messages.py

```
@@ -37,189 +35,230 @@ async def enqueue(payload: dict | list[dict]):
-async def create_batch_messages_for_session(
+@router.post("/", response_model=list[schemas.Message])
+async def create_messages_for_session(
     background_tasks: BackgroundTasks,
-    app_id: str = Path(..., description="ID of the app"),
-    user_id: str = Path(..., description="ID of the user"),
+    workspace_id: str = Path(..., description="ID of the workspace"),
     session_id: str = Path(..., description="ID of the session"),
-    batch: schemas.MessageBatchCreate = Body(
+    messages: schemas.MessageBatchCreate = Body(
         ..., description="Batch of messages to create"
     ),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)